Podfile

```diff
@@ -6,43 +6,45 @@
 
 source 'https://github.com/CocoaPods/Specs.git'
 
-target :lib, :exclusive => true do
-  platform :ios, '7.0'
-  link_with ['Snowplow']
-  pod 'FMDB', '2.5'
-  pod 'Reachability', '3.2'
-end
+abstract_target 'Base' do
 
-target :specs, :exclusive => true do
-  platform :ios, '7.0'
-  link_with ['SnowplowTests']
-  pod 'Nocilla'
-  pod 'SnowplowIgluClient'
-end
+  target 'Snowplow' do
+    inherit! :search_paths
+    platform :ios, '8.0'
+    pod 'FMDB', '2.6.2'
+    pod 'Reachability', '3.2'
+  end
 
-target 'Snowplow-OSX' do
+  target 'Snowplow-OSX' do
+    pod 'FMDB', '2.6.2'
     platform :osx, '10.9'
-    pod 'FMDB', '2.5'
+  end
 end
 
-target 'Snowplow-OSXTests' do
-  platform :osx, '10.9'
-  pod 'Nocilla'
-  pod 'SnowplowIgluClient'
+abstract_target 'BaseTests' do
+
+  target 'SnowplowTests' do
+    inherit! :search_paths
+    platform :ios, '8.0'
+    pod 'Nocilla'
+    pod 'SnowplowIgluClient'
+  end
+
+  target 'Snowplow-OSXTests' do
+    platform :osx, '10.9'
+    pod 'Nocilla'
+    pod 'SnowplowIgluClient'
+  end
 end
 
 post_install do |installer|
-  if Gem::Version.new(Gem.loaded_specs['cocoapods'].version) >= Gem::Version.new('0.38.0')
-    install_38 installer
-  else
-    install_37 installer
-  end
+  install_38 installer
```

Comment:
I am not sure `install_38` means anything anymore as you are using a much newer version of Cocoapods now.  Might be good to rename this function or simply to absorb the code into the `post_install do |installer|` block?